### ISSUE TITLE
Translation please

### ISSUE THREAD

**Issue by thesourcerer8** [mineda-support/ICPS2023_5#1]
Please provide the original files so that we can translate them, the PDFs are a bit harder to work with. Thanks a for publishing the PDK!

**Comment by mineda-support** [comment]
I am afraid it is too early to make translations. Please let me know what you are intended.

**Comment by thesourcerer8** [comment]
I am primarily interested in the DRC rules, I am thinking about doing a standard cell library and test structures with it.

**Comment by mineda-support** [comment]
> I am primarily interested in the DRC rules, I am thinking about doing a standard cell library and test structures with it.

You have answered my question here! DRC rules are not complicated at all. Please take a look at Technology/tech/drc/drc.lydrc.

**Comment by mineda-support** [comment]
> I am primarily interested in the DRC rules, I am thinking about doing a standard cell library and test structures with it.

I should have mentioned that our PDK currently supports 1P1M due to VIA process difficulty. We will release another PDK when 1P2M is available.

**Comment by thesourcerer8** [comment]
Yes, I am aware of that. I will get my process engineer in contact with you to help you with the VIA's. But I also got another request for a 1P1M process node some time ago for tapeout-tapein roundtrip time reasons, so I would like to try it anyway to build some standard cells with it. And I heard that Graywolf has a channel router, so 1P1M might be usable.
